Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/TapToPlace/HideTapToPlaceLabel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.Experimental;
+using Microsoft.MixedReality.Toolkit.Experimental.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities.Solvers;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
```

Comment:
Super nit: Sort using statements alphabetically

Reply:
FYI, right-click on this area of code in Visual Studio and select "Remove and sort usings" to automate this